src/Modules/Drawer.react.js

```diff
@@ -257,23 +257,23 @@ class Drawer extends Component {
         renderContent = React.Children.map(children, child => {
             if (_.isFunction(child.type)) {
                 return React.cloneElement(child, {
-                    closeButton: this.props.closeButton,
-                    inverse: this.props.inverse,
+                    closeButton: closeButton,
+                    inverse: inverse,
                     onClose: this._onClose,
-                    title: this.props.title,
-                    titleTruncate: this.props.titleTruncate,
+                    title: title,
+                    titleTruncate: titleTruncate,
```

Comment:
If I were getting _super_ nit-picky, I'd suggest we can use the short-hand whenever assigning a value to a field of the same name.  e.g. This can be written as:
```JavaScript
return React.cloneElement(child, {
    closeButton,
    inverse,
    title,
    titleTruncate,
    onClose: this._onClose,
});
```

But, meh, we don't have to go that far. :-)